.eslintrc.js

```diff
@@ -460,8 +458,8 @@ module.exports = {
       },
     },
     {
-      'files': ['3p/**/*.js', 'src/**/*.js', 'test/**/*.js', 'testing/**/*.js'],
-      'rules': {'import/order': 2},
+      'files': ['build-system/**/*.js'],
```

Comment:
Is this necessary?  I imagine the rule wouldn't autofix requires in `build-system`, no?

Reply:
Sadly it is! It tries to keep them all in order..but they count as "side effects" and can't be autofixed. Its a catch 22.

Reply:
I'm confused at why `import/order` would be touching `require` statements, but I'm fine with punting

Reply:
B/c `order` operates on `require()` as well: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md#importorder-enforce-a-convention-in-module-import-order